### PR TITLE
Initial drop for mpJwt-2.0 for issue #18531

### PR DESCRIPTION
**Pull request by teddyjtorres**
For issue #18531

### CODE REVIEW

**Comment by teddyjtorres** [comment]
@OpenLiberty/message-reviewer please note that the only update was to create new translation files with the feature version updated from 1.2 to 2.0.

**Comment by teddyjtorres** [comment]
#build

File: dev/com.ibm.ws.security.mp.jwt.cdi/bnd.bnd [hunk 1]
```diff
@@ -6,10 +6,20 @@ Bundle-Name: com.ibm.ws.security.mp.jwt.cdi
...
+jakartaFinalBundleName: Liberty Microprofile JWT CDI
+jakartaFinalBundleId: io.openliberty.security.mp.jwt.cdi.internal
+jakartaFinalBundleDescription: Liberty Microprofile JWT CDI; Jakarta Enabled
+
 Import-Package: \
     !com.ibm.ws.security.mp.jwt.cdi, \
-	javax.enterprise.*; version="[1.1,3)",\
-	*
+    javax.enterprise.*; version="[1.1,3)",\
+    org.eclipse.microprofile.jwt; version="[1.0,3)",\
```
Comment (ayoho) [inline dev/com.ibm.ws.security.mp.jwt.cdi/bnd.bnd:21]: Do you know why this doesn't seem to have been needed before but is now?

File: dev/com.ibm.ws.security.mp.jwt/bnd.bnd [hunk 2]
```diff
@@ -36,6 +45,7 @@ Import-Package: \
    javax.servlet;version="!", \
    javax.servlet.*;version="!", \
    com.ibm.ws.security.jwt.utils, \
+   org.eclipse.microprofile.jwt; version="[1.0,3)", \
```
Comment (ayoho) [inline dev/com.ibm.ws.security.mp.jwt/bnd.bnd:48]: Do you know why this doesn't seem to have been needed before but is now?

**Review by ayoho — APPROVED** [review]

**Comment by teddyjtorres** [comment]
#libby
#build

**Comment by teddyjtorres** [comment]
Last commit has a minor change to set the new features' edition to "full" for their current "noship" kind to fix failures in the com.ibm.ws.feature.tests.VisibilityTest.

**Comment by teddyjtorres** [comment]
The com.ibm.ws.install.jdk.test.SetupJDKTest and com.ibm.ws.ejbcontainer.security.jacc_fat.EJBJarMixM02InWarTest test failures were examined and are not related to the code being delivered.

**Pull request merged**